src/geometry/Bounds.js

```diff
@@ -73,6 +73,18 @@ Bounds.prototype = {
 		return new Point(this.max.x, this.min.y);
 	},
 
+	// @method getTopLeft(): Point
+	// Returns the top-left point of the bounds.
+	getTopLeft: function () {
+		return new Point(this.min.x, this.min.y); // left, top
+	},
+
+	// @method getBottomRight(): Point
+	// Returns the bottom-right point of the bounds.
+	getBottomRight: function () {
+		return new Point(this.max.x, this.max.y); // right, bottom

```

Comment:
Hi,

Thanks for the review and catch!

Now that I realize (:ashamed:) that we just return the same values  as `this.max`, I wonder if we should even clone it at all?

`LatLngBounds` do not bother cloning: https://github.com/Leaflet/Leaflet/blob/master/src/geo/LatLngBounds.js#L100-L110

Cloning is the exact transcription of my initial code.
Doing so is for sure the safest way, but slightly expensive performance-wise (e.g. see https://github.com/Leaflet/Leaflet.markercluster/pull/536#issuecomment-149535377).

Not cloning is the most simple, but could lead to app mistakes if the developer modifies them without cloning first.
But it is entirely consistent with `LatLngBounds`.

Furthermore, as these 2 methods are new, it should not break anything (yet) exposing them with potentially error-prone behaviour, should the app developer be not cautious enough.

For simplicity and consistency with `LatLngBounds`, I assume you would be ok to not clone and directly output `this.max` (same for `this.min`), so I will modify this PR accordingly.

Please feel free to comment back should you have a different position.